### PR TITLE
feat(ci): add badges, add goveralls

### LINKED ISSUE #41: [ci] Add code coverage report in CI
Blocked by #7

### PR DESCRIPTION
**Pull request by Jimexist**
add badges

merge after #54 

fixes #41

### CODE REVIEW

**Comment by gaocegege-caicloud** [comment]
@Jimexist #54 Merged, please rebase and test the PR again :-)

File: README.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
 
 # KubeFlow Controller
 
+[![Build Status](https://travis-ci.org/caicloud/kubeflow-controller.svg?branch=master)](https://travis-ci.org/caicloud/kubeflow-controller)
+[![GoDoc](https://img.shields.io/badge/godoc-reference-blue.svg)](https://godoc.org/github.com/caicloud/kubeflow-controller)
```
Comment (gaocegege) [inline README.md]: 404 not found, I am not sure what happened 🤔

**Comment by gaocegege-caicloud** [comment]
https://travis-ci.org/caicloud/kubeflow-controller/jobs/324104895 hangs, maybe we should re-push and file a new build.

File: README.md [hunk 2]
```diff
@@ -19,17 +20,17 @@ We have a similar project in CaiCloud and it serves for several years. In that p
 
 We separate Kubeflow into two parts:
 
-- Kubeflow controller, which contains the controller for Kuberflow CRD.
-- [Kubeflow clientset][caicloud/kubeflow-clientset], which contains the clientset and CRD specification for Kubeflow.
+* Kubeflow controller, which contains the controller for Kuberflow CRD.
```
Comment (gaocegege) [inline README.md]: Why should we use * instead of -

Comment (Jimexist) [reply README.md]: automatic change from prettier plugin...

Comment (Jimexist) [reply README.md]: vscode-prettier plugin, to be precise

Comment (gaocegege) [reply README.md]: It is hard to type a `*` than `-` since Shift also need to be pressed if you want to type a `*`, but it does not matter :-)

Comment (Jimexist) [reply README.md]: prettier does the job, no typing. 

also prettier fixes table syntax for you in markdown.

**Pull request merged**